runtime/runtime.go

```diff
@@ -1578,32 +1543,45 @@ func (r *interpreterRuntime) meteringInterpreterOptions(runtimeInterface Interfa
 	return []interpreter.Option{
 		interpreter.WithOnStatementHandler(
 			func(_ *interpreter.Interpreter, _ ast.Statement) {
-				checkComputationLimit(1)
+				var err error
+				wrapPanic(func() {
+					err = runtimeInterface.MeterComputation(OpTypeStatement, 1)
```

Comment:
Is there every going to be a situation in which we meter more than one of a computation type at a time? If not IMO we should get rid of the numeric argument here.

Reply:
for some other types like CopyOfArray would be capturing the size of input instead of 1.